mdp/utils/_symeig.py

```diff
@@ -139,8 +140,29 @@ def _symeig_fake(A, B = None, eigenvectors = True, turbo = "on", range = None,
             # diagonalize A
             w, ZA = numx_linalg.eigh(A)
             Z = mdp.utils.mult(ZB, ZA)
-    except numx_linalg.LinAlgError as exception:
-        raise SymeigException(str(exception))
+    except:
```

Comment:
What exceptions are to be expected here? I guess this is something like cupy is not supported. In that case mdp souldn't silently fallback to using numpy, but tell the user the issue (an maybe recommend using a different numx backend).

Reply:
https://realpython.com/the-most-diabolical-python-antipattern